src/renderer/components/ft-video-player/ft-video-player.vue

```diff
@@ -7,6 +7,7 @@
       controls
       preload="auto"
       :data-setup="JSON.stringify(dataSetup)"
+      crossorigin="anonymous"
```

Comment:
What's the use of this change?

Reply:
If API is set to Invidious and video is in legacy format, canvas will be "tainted"
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-crossorigin